app/src/main/java/com/jerboa/ui/components/comment/CommentNodes.kt

```diff
@@ -58,6 +58,7 @@ fun CommentNodes(
     showAvatar: Boolean,
     blurNSFW: Int,
     showScores: Boolean,
+    swipeToActionPreset: Int,
```

Comment:
Don't pass the int, pass a simple settings preset enum around. You should only need to convert to the to / from ordinal at the DB layer.

Reply:
The problem is we kinda don't do that for any of the enums.

Reply:
> Don't pass the int, pass a simple settings preset enum around. You should only need to convert to the to / from ordinal at the DB layer.

If we do it in DB layer, it will mean that we'll need to use strings to store enums because AppSettings entity is directly used everywhere. I've seen that Ints are mostly used for that kind of stuff in Jerboa so won't it make it look a little inconsistent?

Or i can simply write a typeconverter

Reply:
> Don't pass the int, pass a simple settings preset enum around. You should only need to convert to the to / from ordinal at the DB layer.

Or i can simply do it like this: 
```
HomeActivity(
                            appState = appState,
                            .......
                            .......
                            postActionbarMode = appSettings.postActionbarMode,
                            swipeToActionPreset = SwipeToActionPreset.entries[appSettings.swipeToActionPreset],
                        )
```

Reply:
* You can also do `appSettings.swipeToActionPreset.toEnum()`
*

Reply:
> toEnum

Oh, that's cool, didn't think too much about this inline+reified approach. Found out it already exists in the code so used it